README.md

```diff
@@ -161,6 +161,12 @@ It is highly recommended that you use Amazon S3. The reasons are:
 
 Why then not require S3? Because while some applications using node-pre-gyp need to distribute binaries as large as 20-30 MB, others might have very small binaries and might wish to store them in a GitHub repo. This is not recommended, but if an author really wants to host in a non-S3 location then it should be possible.
 
+It is also possible to use an s3-compatible storage backend, such as minio, as well as non-standard s3 endpoints. In this case, you can specify the following properties in the `binary` section:
```

Comment:
Can you add a little more detail around what kind of errors or situations a user might hit such that they would need to provide these options?

Reply:
What about:
If you are not using a standard s3 path like `bucket_name.s3(.-)region.amazonaws.com`, you might get an error on `publish` because node-pre-gyp extracts the region and bucket from the `host` url. For example, you may have an on-premises s3-compatible storage server, or may have configured a specific dns redirecting to an s3 endpoint. In these cases, you can explicitely set the `region` and `bucket` properties to tell node-pre-gyp to use these values instead of guessing from the `host` property. The following values will be used in the binary section:

Reply:
👍